en/manage-cli-interfaces.md

```diff
@@ -0,0 +1,223 @@
+Title: CLI Interface Management
+table_of_contents: True
+
+
+# CLI Interface Management
+
+This is a list of interface management tasks to perform with the MAAS CLI. See
```

Comment:
surely "which can be performed"?

Reply:
I'll change this, thanks. It's the same as other CLI docs; I kept it to remain consistent with the active voice used throughout.